zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/mutable/MutableUserState.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.engine.state.mutable;
+
+import io.camunda.zeebe.engine.state.immutable.UserState;
+import io.camunda.zeebe.protocol.impl.record.value.user.UserRecord;
+import org.agrona.DirectBuffer;
+
+public interface MutableUserState extends UserState {
+
+  void create(final UserRecord user);
+
+  void update(final UserRecord user);
+
+  void delete(final DirectBuffer username);
+
+  void delete(final String username);
```

Comment:
❌ All these methods are unused and untested. I'd suggest either removing them if you don't need them yet, or if you decide to keep them to properly test these methods.